frontend/packages/console-shared/src/utils/operator-utils.ts

```diff
@@ -0,0 +1,40 @@
+import * as _ from 'lodash';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager/src';
+import { CAMEL_SOURCE_INTEGRATION } from '@console/knative-plugin';
+import { TopologyDataResources } from '@console/dev-console/src/components/topology';
+import { getDynamicEventSourcesModelRefs } from '@console/knative-plugin/src/utils/fetch-dynamic-eventsources-utils';
+import { TopologyResources } from '../types';
```

Comment:
`console-shared` should not be using functions from `knative-plugin` nor types `dev-console`